src/ray/common/ray_config_def.h

```diff
@@ -187,13 +187,19 @@ RAY_CONFIG(int, object_manager_push_timeout_ms, 10000)
 
 /// The period of time that an object manager will wait before pushing the
 /// same object again to a specific object manager.
-RAY_CONFIG(int, object_manager_repeated_push_delay_ms, 60000)
+/// NOTE(ekl): this has been raised to avoid spurious repeated pushes.
+RAY_CONFIG(int, object_manager_repeated_push_delay_ms, 10 * 60 * 1000)
```

Comment:
Will be removed entirely in a follow-up https://github.com/ray-project/ray/pull/11874/files